lib/nrf_modem_lib/lte_connectivity/lte_connectivity.h

```diff
@@ -42,6 +42,18 @@ enum lte_connectivity_options {
 	LTE_CONNECTIVITY_IF_DOWN = 1,
 };
 
+
+struct lte_connectivity_state {
```

Comment:
Looks like this type is only needed in one file, so could optionally be moved from the header file